### DISCUSSION TITLE
User/Roles configuration

### DISCUSSION THREAD

**Discussion by Fvegini** [mlflow/mlflow#7893] [Q&A] [answered]
In MLFlow documentation there's some reference to create projects with access control, but I can't find how to do it.

When I open my mlflow UI I can't see anything related to authentication;

It is really possible to do this with MLFlow or would be needed to buy a Iguazio licence to do it?

**Comment by JHibbard** [comment, marked as answer]
Hi Fvegini,

Could you share a link to the MLflow documentation you found for creating projects with access control? I don't believe that's a feature of OSS MLflow. The Databricks managed MLflow has access control functionality as do other managed MLflow services from other providers.

Hope that helps!
Jim
